src/invoices/InvoiceDetails/ApproveConfirmationModal/useDuplicateInvoice.js

```diff
@@ -0,0 +1,63 @@
+import { useEffect, useState } from 'react';
+import moment from 'moment';
+import { keyBy } from 'lodash';
+
+import {
+  batchFetch,
+  LIMIT_MAX,
+  DATE_FORMAT,
+  useShowCallout,
+} from '@folio/stripes-acq-components';
+
+const getInvoices = async (mutator, invoice) => {
+  const formattedInvoiceDate = moment.utc(invoice.invoiceDate)?.format(DATE_FORMAT);
+
+  try {
+    return await mutator.duplicateInvoices.GET({
+      limit: `${LIMIT_MAX}`,
+      params: {
+        query: `id<>"${invoice.id}" AND
+          vendorInvoiceNo=="${invoice.vendorInvoiceNo}" AND
+          invoiceDate=="${formattedInvoiceDate}*" AND
+          vendorId=="${invoice.vendorId}"`,
+      },
+    });
+  } catch {
+    throw new Error('ui-invoice.invoice.actions.load.error');
```

Comment:
I'm not sure using `Error` is necessary here.
We only guard an API call here, not the bunch of code.
This would be the same, without Error, we return just the reason for our own use:
```js
mutator.duplicateInvoices.GET(...)
  .catch(() => Promise.reject('ui-invoice.invoice.actions.load.error'))
```

Reply:
In both cases (`Promise.reject` and `throw`) we have to use `new Error` to avoid  `ESLint` warnings